tools/wolfram.py

```diff
@@ -0,0 +1,22 @@
+import wolframalpha
+
+from config import WOLFRAMALPHA_API
+
+client = wolframalpha.Client(WOLFRAMALPHA_API)
+
+
+def get_wolfram_result(question: str):
+    res = client.query(question)
+    answer = ""
+    for pod in res.pods:
+        pod_title = pod.title + ":\n"
+        subpod_text = ""
+        for sub in pod.subpods:
+            if sub.plaintext is not None:
+                subpod_text += str(sub.plaintext)
+                subpod_text += "\n"
+
+        if subpod_text:
+            answer += pod_title + subpod_text
+            answer += "\n"
```

Comment:
with f-string it might be more readable in this case

`subpod_text += f"{sub.plaintext}\n"`
and 
`answer += f"{pod_title}{subpod_text}\n"`